### PR TITLE
add videomail-client as a dependency

### PR DESCRIPTION
**Pull request by kjohnson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "ninja-forms-videomail",
   "version": "1.0.0",
+  "dependencies": {
+    "videomail-client": "1.11.0"
```
Comment (binarykitchen) [inline package.json]: awesome :)

File: includes/Fields/Videomail.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ public function admin_form_element( $id, $value )
 
     public function enqueue_scripts()
     {
-        wp_enqueue_script( 'nf-videomail-client', NF_Videomail::$url . '/assets/js/lib/videomail-client.min.js', false );
+        wp_enqueue_script( 'nf-videomail-client', NF_Videomail::$url . '/node_modules/videomail-client/dist/videomail-client.min.js', false );
         wp_enqueue_script( 'nf-videomail', NF_Videomail::$url . '/assets/js/min/videomail.min.js', array( 'nf-front-end', 'nf-videomail-client' ) );
```
Comment (binarykitchen) [inline includes/Fields/Videomail.php]: what about this?

Comment (kjohnson) [inline includes/Fields/Videomail.php]: That is the minified JS for the custom videomail Ninja Forms Field controller. (created with gulp)

Comment (binarykitchen) [inline includes/Fields/Videomail.php]: oh right, my bad, thanks

**Pull request merged**